Dockerfile

```diff
@@ -45,6 +45,10 @@ ENV TORCH_CUDA_ARCH_LIST="6.0;6.1;7.0;7.5;8.0;8.6;8.9;9.0+PTX"
 COPY . /tmp/app
 RUN pip install /tmp/app && rm -rf /tmp/app
 
+RUN git clone -b feat/alibi https://github.com/smallcloudai/flash-attention.git /tmp/flash-attention \
+    && cd /tmp/flash-attention \
+    && MAX_JOBS=8 python3 setup.py install
+
```

Comment:
we should add flash_attn in setup, something like this
```
"flash_attn @ git+https://github.com/smallcloudai/flash-attention@feat/alibi"
```

to build it in docker you need some env vars:
```
ENV MAX_JOBS=8
ENV FLASH_ATTENTION_FORCE_BUILD="TRUE"
```

I tested this in docker image but not with docker build. Hope it'll help!

Reply:
So, have you succeeded in building the necessary version just with `flash_attn @ git+https://github.com/smallcloudai/flash-attention@feat/alibi` ? Or you're suggesting to keep them both in the docker file and `setup.py`?